keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -159,26 +157,6 @@ internal class ApplicationControllerTests : JUnit5Minutests {
               ),
               _defaults = clusterDefaults
             )
```

Comment:
Moved a lot of these tests out of this class, but the actual tests stayed the same in `ApplicationServiceTests`